<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>582</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>582</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>582</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MusicPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
 bord er:none
  
}

QPushButton:hover
{
	
	
	background-color: rgb(255, 239, 233);
}

</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>991</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>64</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230,230,230,0.8);
border-radius:25px;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>29</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/folder.png);
background-color:rgba(0,0,0,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="13">
     <widget class="QPushButton" name="pushButton_5">
      <property name="minimumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/volume-up.png);
background-color:rgba(0,0,0,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <spacer name="horizontalSpacer_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>88</width>
        <height>35</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/rewind.png);
background-color:rgba(0,0,0,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QPushButton" name="pushButton_4">
      <property name="minimumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/forward.png);
background-color:rgba(0,0,0,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>58</width>
        <height>29</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>38</width>
        <height>35</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/play.png);
background-color:rgba(0,0,0,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="11">
     <widget class="QPushButton" name="pushButton_Theme">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/2740606.png);
background-color:rgba(0,0,0,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <spacer name="horizontalSpacer_13">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>58</width>
        <height>29</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="10">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>38</width>
        <height>35</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>118</width>
        <height>29</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton_6">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="14">
     <spacer name="horizontalSpacer_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>29</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>331</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255,0.5);
border-radius:10px;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>1021</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QSlider" name="playCourseSlider">
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
border: 0px solid #bbb;
}

/*1.滑动过的槽设计参数*/
QSlider::sub-page:horizontal {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(16, 181, 140, 255), stop:1 rgba(255, 255, 255, 255));
border-radius: 2px;
margin-top:8px;
margin-bottom:8px;
}

/*2.未滑动过的槽设计参数*/
QSlider::add-page:horizontal {
background: rgb(255,255, 255);
border: 0px solid #777;
border-radius: 2px;
margin-top:9px;
margin-bottom:9px;
}
 
/*3.平时滑动的滑块设计参数*/
QSlider::handle:horizontal {
/*滑块颜色*/
background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 191, 0, 255), stop:1 rgba(255, 255, 255, 255));
/*滑块的宽度*/
width: 5px;
border: 1px solid rgb(193,204,208);
border-radius: 2px; 

margin-top:4px;

margin-bottom:4px;
}

/*4.手动拉动时显示的滑块设计参数*/
QSlider::handle:horizontal:hover {

background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 191, 0, 255), stop:1 rgba(255, 255, 255, 255));
width: 10px;
border: 1px solid rgb(193,204,208);
border-radius: 5px; 
margin-top:2px;
margin-bottom:2px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="curLabel">
         <property name="styleSheet">
          <string notr="true">color:rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>696</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="totalLabel">
         <property name="styleSheet">
          <string notr="true">color:rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_volume">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>450</y>
     <width>35</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(59, 59, 59);
border-radius:12px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_volume">
      <property name="sizeIncrement">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSlider" name="Slider_volume">
      <property name="styleSheet">
       <string notr="true">QSlider::groove:vertical {
border: 0px solid #bbb;

}

QSlider::sub-page:vertical {
background:  rgb(255, 255, 255);

border-radius: 0px;/*整体圆角*/
border-top-left-radius:10px;/*左上圆角*/
border-bottom-left-radius:10px;/*左下圆角*/
margin-top:0px;/*上距离*/
margin-bottom:0px;/*下距离*/
}

QSlider::add-page:vertical {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(16, 181, 140, 255), stop:1 rgba(255, 255, 255, 255));/*背景色*/
border-radius: 0px;/*整体圆角*/
border-top-right-radius:10px;/*右上圆角*/
border-bottom-right-radius:10px;/*右下圆角*/
margin-top:0px;/*上距离*/
margin-bottom:0px;/*下距离*/
}
/*拖拽键样式*/
QSlider::handle:vertical {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 191, 0, 255), stop:1 rgba(255, 255, 255, 255));
border: 0px solid rgb(255,153,102);
width: 14px;
height:10px;
border-radius: 7px;
margin-top:0px;
margin-bottom:0px;
}

QSlider::sub-page:vertical:disabled {
 
}

QSlider::add-page:vertical:disabled {
 
}
QSlider::handle:vertical:disabled {
 
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
